main.go

```diff
@@ -184,6 +206,14 @@ func main() {
 					},
 				},
 			},
+			{
+				Name:     "renew-root",
+				HelpName: "renew-root <EST Server> [<rotation-id>]",
```

Comment:
It looks like inconcistency between the cli parameter name `rotation-id` and the state field `handler.State.CorrelationId`. I suppose that's because it comes from the base class.

Reply:
I wanted this to be `correlation-id`, but it must be `rotation-id` for backward compatibility (required for `renew-cert`).
So, in [this PR](https://github.com/foundriesio/fioconfig/pull/48/commits/e0b681c760c4eca7ea911840b1fd1743423c99db#diff-1b8a337f0d83d790b4c9fddefa1930f7b629cd8370c1b58fa05a5e984def11cdR19) I ended up renaming the runtime field name, but leaving the state file property on disk the old way.

That said, I tend to agree with you that the command-line argument needs to be renamed too, as it is a positional argument, especially for this new command.